nucypher/policy/conditions/evm.py

```diff
@@ -56,16 +56,6 @@
 # Permitted blockchains for condition evaluation
 from nucypher.utilities import logging
 
-_CONDITION_CHAINS = {
-    1: "ethereum/mainnet",
-    11155111: "ethereum/sepolia",
-    137: "polygon/mainnet",
-    80002: "polygon/amoy",
-    # TODO: Permit support for these chains
-    # 100: "gnosis/mainnet",
-    # 10200: "gnosis/chiado",
-}
-
```

Comment:
Right.  This was the first iteration of limiting which chains are permitted for mainnet condition evaluation - which has now been replaced partly by nucpher/chainlist domain-named files.  Does this also imply that there are no longer any restrictions placed on user-defined endpoints as well?

Reply:
this PR doesn't add functionality for user defined endpoints

Reply:
Clarifying what you mean by "user".

If you mean restrictions on "node providers", that still holds for `mainnet`/`tapir` - see https://github.com/nucypher/nucypher/pull/3569/commits/da23d687bef0d23064b16aaae64c8865dfc3ea6b although `user_configured_endpoints` may be a misnomer there, perhaps it should be `node_configured_endpoints`... cc @theref .

If you mean the user that defines the conditions, the `nucypher/chainlist domain-named files` is what replaces user-defined endpoints in conditions. Instead, public rpc endpoints are used for whatever chain ID outside of ethereum/polygon is specified. It's part of the reason we are only allowing on `lynx` so that we can understand if this is sufficient or not. Developer prototyping can happen without the condition itself needing to hardcode an RPC URL.

Reply:
Fixed name of `user_configured_endpoints` variable, and associated log/exception messages - https://github.com/nucypher/nucypher/pull/3569/commits/9494ede1716686e3def775d069d66e82c93c8e64